src/nacl/bindings/crypto_generichash.py

```diff
@@ -122,6 +122,26 @@ def generichash_blake2b_salt_personal(data,
     return ffi.buffer(digest, digest_size)[:]
 
 
+class Blake2State(object):
+    """
+    Python-level wrapper for the crypto_generichash_blake2b state buffer
+    """
+    __slots__ = ['statebuf', 'digest_size']
+
+    def __init__(self, digest_size):
+        self.statebuf = ffi.new("unsigned char[]",
```

Comment:
This should be underscore prefixed since a `Blake2State` instance should be considered opaque outside of the `digest_size`.

Reply:
Will change